pages/common/jhipster.md

```diff
@@ -0,0 +1,36 @@
+# jhipster
+
+> Generates a monolithic or microservices project after prompting user.
+> More information: <https://www.jhipster.tech/>.
+
+- Generate a simple full-stack project (monolithic or microservices):
+
+`jhipster`
+
+- Generate a simple frontend project (with OPTION = --skip-server) or a simple backend project (with OPTION = --skip-client):
+
+`jhipster [OPTION]`
```

Comment:
```suggestion
- Generate a simple frontend project with `--skip-server` or a simple backend project with `--skip-client`:

`jhipster {{option}}`
```
It might be simple and clear (although I would wait for the review by another one).